services/nublado2/values.yaml

```diff
@@ -0,0 +1,483 @@
+        dask_worker.yml: |
+          {{ dask_yaml | indent(6) }}
+    # When we break out the resources we should make this per-instance
+    #  configurable.
+    - apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: idds-config
+        namespace: "{{ user_namespace }}"
+      data:
+        idds_cfg.client.template: |
+          # Licensed under the Apache License, Version 2.0 (the "License");
+          # You may not use this file except in compliance with the License.
+          # You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+          #
+          # Authors:
+          # - Wen Guan, <wen.guan@cern.ch>, 2020
+          [common]
+          # if logdir is configured, idds will write to idds.log in this directory.
+          # else idds will go to stdout/stderr.
+          # With supervisord, it's good to write to stdout/stderr, then supervisord can manage and rotate logs.
+          # logdir = /var/log/idds
+          loglevel = INFO
+          [rest]
+          host = https://iddsserver.cern.ch:443/idds
+          #url_prefix = /idds
+          #cacher_dir = /tmp
+          cacher_dir = /data/idds
+    - apiVersion: v1
+      kind: ServiceAccount
+      metadata:
+        name: "{{ user }}-serviceaccount"
+        namespace: "{{ user_namespace }}"
+      imagePullSecrets:
+      - name: pull-secret
+    - apiVersion: rbac.authorization.k8s.io/v1
+      kind: Role
+      metadata:
+        name: "{{ user }}-role"
+        namespace: "{{ user_namespace }}"
+      rules:
+      # cf https://kubernetes.dask.org/en/latest/kubecluster.html
+        - apiGroups: [""]
+          resources: ["pods", "services"]
+          verbs: ["create", "delete", "get", "list", "watch"]
+        - apiGroups: [""]
+          resources: ["pods/log"]
+          verbs: ["get","list"]
+        - apiGroups: ["policy"]
+          resources: ["poddisruptionbudgets"]
+          verbs: ["create", "delete", "get", "list", "watch"]
+    - apiVersion: rbac.authorization.k8s.io/v1
+      kind: RoleBinding
+      metadata:
+        name: "{{ user }}-rolebinding"
+        namespace: "{{ user_namespace }}"
+      roleRef:
+        apiGroup: rbac.authorization.k8s.io
+        kind: Role
+        name: "{{ user }}-role"
+      subjects:
+        - kind: ServiceAccount
+          name: "{{ user }}-serviceaccount"
+          namespace: "{{ user_namespace }}"
+    - apiVersion: ricoberger.de/v1alpha1
+      kind: VaultSecret
+      metadata:
+        name: butler-secret
+        namespace: "{{ user_namespace }}"
+      spec:
+        path: "{{ butler_secret_path }}"
+        type: Opaque
+    - apiVersion: ricoberger.de/v1alpha1
+      kind: VaultSecret
+      metadata:
+        name: pull-secret
+        namespace: "{{ user_namespace }}"
+      spec:
+        path: "{{ pull_secret_path }}"
+        type: kubernetes.io/dockerconfigjson
+
+# Note: See note above about existingSecret.
+vault_secret_name: "nublado2-secret"
```

Comment:
May as well just hard-code this in the resource since we don't ever override it.

Reply:
OK.